solutions/cf-1208F.mdx

```diff
@@ -0,0 +1,74 @@
+---
+id: cf-1208F
+source: CF
+title: Bits and Pieces
+author: Dong Liu
+---
+
+## Explanation
+
+Let $|$ denote the bitwise OR operation and $\&$ denote the bitwise AND operation.
+
+The answer is equivalent to enumerating $a_i$ and finding the optimal $x$ where $x=a_j\&a_k$ and $i<j<k$ and $a_i|x$ is maximum.
```

Comment:
maybe smth like "equivalent to finding an $x$ such that [asdf] and $a_i|x$ is maximized"
also what does "enumerating" mean?

Reply:
> what does "enumerating" mean?

like looping over

Reply:
actually i'd remove the enumerating part, doesn't seem to add anything

Reply:
how about replacing "enumerating" with "fixing an"?

Reply:
why do we have to fix $i$ if it's going to go over all $i$ anyways
$i$, $j$, and $k$ can be basically any indices from what i'm seeing in this statement

Reply:
ohh I see